controllers/api.js

```diff
@@ -140,19 +135,16 @@ exports.getFacebook = function(req, res, next) {
  * GET /api/scraping
  * Web scraping example using Cheerio library.
  */
-exports.getScraping = function(req, res, next) {
-  cheerio = require('cheerio');
-  request = require('request');
-
-  request.get('https://news.ycombinator.com/', function(err, request, body) {
-    var $ = cheerio.load(body);
-    var links = [];
-    $('.title a[href^="http"], a[href^="https"]').each(function() {
+exports.getScraping = (req, res, next) => {
+  request.get('https://news.ycombinator.com/', (err, request, body) => {
+    const $ = cheerio.load(body);
+    const links = [];
+    $('.title a[href^="http"], a[href^="https"]').each(() => {
```

Comment:
I just refactored `api.js` myself, before I saw this PR.  
  
There's one problem I spotted right away, that I encountered in my own refactoring.

In the web scraping API, this code will not work as you expect.
```
$('.title a[href^="http"], a[href^="https"]').each(() => {
      links.push($(this));
    });
```
If you try logging `this`, you'll see that it now refers to the global `this`.
Previously, cheerio's `each` function sets `this` in the regular callback function to the current element.  
However, with the arrow callback function, `this` is bound lexically so that has no effect.
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Invoked_through_call_or_apply

You'll need to change this to:
```
$('.title a[href^="http"], a[href^="https"]').each((i, elem) => {
      links.push($(elem));
    });
```